nise/__main__.py

```diff
@@ -226,12 +234,46 @@ def _validate_provider_inputs(parser, options):
     return (valid_inputs, provider_type)
 
 
+def _load_yaml_file(filename):
+    """Local data from yaml file."""
+    yamlfile = None
+    if filename:
+        try:
+            with open(filename, 'r+') as yaml_file:
+                yamlfile = yaml.load(yaml_file)
+        except TypeError:
+            yamlfile = yaml.load(filename)
+    return yamlfile
+
+
+def _load_static_report_data(parser, options):
+    """Validate/load and set start_date if static file is provided."""
+    if options.get('static_report_file'):
+        start_dates = []
+        if options.get('start_date'):
+            start_date_argument = options.get('start_date')
+            start_dates.append(start_date_argument)
+        static_report_data = _load_yaml_file(options.get('static_report_file'))
```

Comment:
Does this work only for relative paths too?

Reply:
yes, relative paths work
```
(nise_fork-lW4EhP1Y) ➜  nise_fork git:(static_reports) nise --aws --start-date 1-1-2019 --static-report-file tests/aws_static_report.yml
(nise_fork-lW4EhP1Y) ➜  nise_fork git:(static_reports)
```